tests/std/tests/P2387R3_bind_back/test.cpp

```diff
@@ -0,0 +1,216 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
Note: this is pretty close to a straight copy of `P0356R5_bind_front`, which is great.

No change requested.